common-compose/src/main/java/com/amity/socialcloud/uikit/common/ui/elements/AmityExpandableTextView.kt

```diff
@@ -126,7 +128,8 @@ fun AmityExpandableText(
         val layoutResult = remember { mutableStateOf<TextLayoutResult?>(null) }
         val pressIndicator = Modifier.pointerInput(Unit) {
             detectTapGestures(
-                onPress = { offset ->
+                // noom update
```

Comment:
leaving this comment as a marker where we made a change for when we need to resolve merge conflicts when updating the fork